Library/Homebrew/cmd/upgrade.rb

```diff
@@ -95,6 +100,12 @@ def upgrade
     end
   end
 
+  def prompt(default, *args)
```

Comment:
Stick this function somewhere in `utils` and making it always prompt for `Y/n` and return a boolean.